index.html

```diff
@@ -241,6 +241,58 @@ <h2 class="about-title">About me</h2>
             </ul>
         </div>
     </section>
+    <section class="form-container">
+        <h3 class="form-heading">
+            I'm always interested in hearing about new projects, 
+            so if you'd like to chat please get in touch.
+        </h3>
+        <form action="https://formspree.io/f/xayvbrnw" method="post" class="form">
+            <ul class="form-inputs">
+                <li class="name-item">
+                    <input class="input" type="text" placeholder="Full name" name="full-name" maxlength="30" required>
+                    <div class="input-devider"></div>
+                </li>
+                <li class="email-item">
+                    <input class="input" type="email" placeholder="Email address" name="email" required>
+                    <div class="input-devider"></div>
+                </li>
+                <li class="comment-item">
+                    <textarea class="input-area" placeholder="Enter text here" name="comments" maxlength="500" required></textarea>
```

Comment:
- [ ] Please, consider the background of your textarea - regarding Figma template it should be `#FBF8F7`